### DISCUSSION TITLE
How do I check the field for fullness of text? (if the field is empty, reset the loop to the beginning)

### DISCUSSION THREAD

**Discussion by DEVAvia** [AutomaApp/automa#1376] [Q&A] [answered]
### How do I check the field for fullness of text? _(if the field is empty, reset the loop to the beginning)_
The site on which I automate has a glitch — sometimes when cloning a template that needs minimal editing, an error occurs due to which the site opens the same page instead of a template with filled fields, but with empty fields that are NOT filled, however, if the automatic creation process continues, then I get the wrong values that are needed in the fields.

Please tell me how I can make a check so that if the specified text field loads empty (without information), so that the further execution of the cycle is interrupted and reset to the beginning (opening the template page and re-cloning to a new page).

**Comment by Delusionnnn** [comment, marked as answer]
Conditions block will be your friend here. You can mess around with different ones here but mostly "Element Text" Equals or Not Equals Value. 

So for example if the text field shows up blank and you want to wait for it to load a value before continuing or have it do some other task then it would look like this: (Element Text and then select your element or if your block requires a Forms > Get Form Value to grab the text then set it up with the variable instead)

![image](https://github.com/AutomaApp/automa/assets/141815049/6e741e26-44a2-42d1-a14d-0d3130afb5f4)

Sometimes you need to put in a space, it just depends on what the text area has as a default. Just gotta play around with it.

This is what it looks like if you need to use the Forms > Get Form Value to grab the text

![image](https://github.com/AutomaApp/automa/assets/141815049/60363b5d-00f1-41c0-bd7b-6eb395866339)
![image](https://github.com/AutomaApp/automa/assets/141815049/e117b4ae-44c8-4093-be75-a6af28c7c688)


Hope this helps! I can help more if needed. Conditions blocks really do a lot in this program.
